onnxruntime/test/contrib_ops/tokenizer_test.cc

```diff
@@ -1,6 +1,8 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-
+#if ((__cplusplus >= 201703L) || (defined(_MSVC_LANG) && (_MSVC_LANG >= 201703L)))
+//TODO: handle the u8string.
+#else
```

Comment:
curious, when is the else case in effect?

Reply:
It was to bypass some build errors in C++20(not 17).